robolectric-utils/src/main/java/org/robolectric/internal/bytecode/InvokeDynamic.java

```diff
@@ -0,0 +1,21 @@
+package org.robolectric.internal.bytecode;
+
+public class InvokeDynamic {
+  public static final boolean ENABLED = useInvokeDynamic();
+
+  private static final String ENABLE_INVOKEDYNAMIC = "robolectric.invokedynamic.enable";
```

Comment:
Should this be documented somewhere? Since currently there is no way to find this variable other that reading the source (and knowing where to look) :)